src/api/spec/factories/users.rb

```diff
@@ -5,10 +5,29 @@
     sequence(:login) { |n| "user_#{n}" }
     password { 'buildservice' }
 
+    transient do
+      avoid_home_project { true }
+    end
+
     trait :in_beta do
       in_beta { true }
     end
 
+    trait :with_home do
+      avoid_home_project { false }
+    end
+
+    to_create do |user, evaluator|
+      before = Configuration.allow_user_to_create_home_project
+      if evaluator.avoid_home_project
+        Configuration.update(allow_user_to_create_home_project: false)
```

Comment:
Why don't you make this default in the suite setup, instead of changing it here all the time?

Reply:
You can still have 2 users in the same test - one with and one without home.

Reply:
Sure and the one `:with_home` can change the default during creation, no?